bundles/XliffBundle/src/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php

```diff
@@ -146,7 +146,7 @@ protected function addLocalizedFields(DataObject\Concrete $object, AttributeSet
         return $this;
     }
 
-    protected function addBlocksInLocalizedfields(Localizedfields $fd, Data $definition, DataObject\Concrete $object, AttributeSet $result, array $exportAttributes = null): void
+    protected function addBlocksInLocalizedfields(Data $definition, DataObject\Concrete $object, AttributeSet $result): void
```

Comment:
I understand your intention of removing unused params but this could lead to BC Break if the service is extended/decorated.

Reply:
@dnlbslcm please see :blush:

Reply:
Sorry, first day back from holidays... :)
I understand the potential problem and reverted the method to the old attribute set.